addons/popochiu/engine/objects/graphic_interface/components/dialog_text/dialog_text.gd

```diff
@@ -70,7 +70,14 @@ func _input(event: InputEvent) -> void:
 
 #region Public #####################################################################################
 func play_text(props: Dictionary) -> void:
-	var msg: String = E.get_text(props.text)
+	var display_text: String = ""
+	
+	if PopochiuConfig.should_talk_gibberish():
+		display_text = D.create_gibberish(props.text)
+	else:
+		display_text = props.text
+	
+	var msg: String = E.get_text(display_text) #E.get_text(props.text)
```

Comment:
Not sure this makes sense.

Isn't it better to gibberish the result of `E.get_text(props.text)` instead?

Reply:
@stickgrinder is right. The result of `E.get_text` is the translation of the **String**. In the future, when we include support for gettext (PO) the thing will probably take the whole line into account to find the proper translation.